src/rules/completedDocsRule.ts

```diff
@@ -59,9 +59,10 @@ export class Rule extends Lint.Rules.TypedRule {
         const options = this.getOptions();
         const completedDocsWalker = new CompletedDocsWalker(sourceFile, options, langSvc.getProgram());
 
-        const nodesToCheck = this.getNodesToCheck(options.ruleArguments);
-        completedDocsWalker.setNodesToCheck(nodesToCheck);
-
+        if (options.ruleArguments !== undefined) {
```

Comment:
You can omit all those checks for `ruleArguments !== undefined` if you make ruleArguments a required property in IOptions. AbstractRule initializes it to an empty array anyways. IIRC the only place where it is omitted is the constructor of EnableDisableRulesWalker in linter.ts